### DISCUSSION TITLE
All about Lobsters! Mechanics, Ideas, looking for Feedback!

### DISCUSSION THREAD

**Discussion by SirSquidly** [SirSquidly/Oceanic-Expanse#53] [Ideas]
I plan on adding Lobsters to the mod, and instead of silently dropping it, I'd like to actually get feedback on it and test out ideas here.

Lobsters are going to be a new Ocean Crustacean _(Which do count as Arthropods, yay for that enchantment)_ that exist to help add some life to the Seafloor, and add a reliable mob for actual food farming. I wanted an Ocean Floor mob as I feel like the addition of Kelp and Seagrass brings attention to the floor, which is very barren of life.

Lobsters have a unique mechanic of growth, where they will actually grow bigger overtime, and molt to shrink back down in size. Molting drops Lobster Shells, which can be used to make Coquina, a new Soup, or directly into Bonemeal.  There will also be some way to stop it from Molting, so you can get mega-sized Lobsters.

Killing a Lobster drops Lobster Meat, which likely will match Steak in terms of restoration,

Another unique mechanic are rare Lobster variants, such as blue, yellow, calico, albino, and the very rare Half-Half lobsters. Half Half lobsters, along with having a split of two different variants, also have the unique mechanic of solo breeding. This is inspired by real life half-half lobsters. Yes, really, they can do that.

Here's a WIP brown Lobster, the most common type:
<img src='https://github.com/SirSquidly/Oceanic-Expanse/assets/13826553/2da7e138-bc7d-4ffe-9baf-dbd60572774b' width='300'>

Here are a few mechanics I am still considering with Lobsters:

1. Maybe you can tame a Lobster? I think it might be funny if they literally can be a pet, and even attack other mobs like a Wolf!

2. Maybe Lobsters have some way to be 'semi-tamed' like Axolotl, where they will defend you, but only for a bit. this kinda matches how Social they are in real life, and might be better than full taming.

3. Maybe allow Lobster Shells to be craftable for some kind of back-dodge item? In real life Lobsters can back-dodge, so it might be funny to allow players to mimic that ability. I worry it's a bit powerful for just Lobsters to give though.

**Comment by Sethgno** [comment]
i think itd be cool to tame and saddle a lobster for an underwater mount! (although i guess that depends on model/size)
and to the third point, i feel like a dodge mechanic ( IMO, :3 ) feels a bit out of scope depending on how its implimented, but in general i like the idea of more armor that adds underwater mobility

**Comment by Sylsatra** [comment]
I think it would be cool if you could implement the second option into Minecraft but with the addition that a semi-tamed Lobster will only protect the area around its burrow (or cave).

**Reply by SirSquidly** [reply]
I'll probably go with this one, as I like the idea a Lobster will defend you... but they aren't gonna follow at all. So they act like a home defense!
